### DISCUSSION TITLE
曼波~

### DISCUSSION THREAD

**Discussion by dwb2012** [share121/inter-knot#2513] [其他]
https://www.bilibili.com/video/BV1JS421o7pM/?spm_id_from=333.337.search-card.all.click&vd_source=5594f8b7af6ddc631ee74fe632d16ebf
![BF029453B7FB95D1AD0FDD2508A4BD98](https://github.com/user-attachments/assets/8d89cd04-42db-4fe0-8d65-dcd800acb3e3)

**Comment by hua-yuanshuai** [comment]
曼波<img src="https://img-static.mihoyo.com/communityweb/upload/3b2e51180d866548d11c650a2dcf6d43.png" alt="三月七_暗中观察" width="75" >
